packages/golden_toolkit/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.2.2
+
+Added `deviceSetupBetweenSizeChanges` to multiscreenGolden to allow customization.  Needed if Images will take several pumps before a widget wtih an image is put into the tree.
```

Comment:
```suggestion
Added `deviceSetupBetweenSizeChanges` to multiscreenGolden to allow pumping customization when device configuration (ex: screen size) changes
```

Reply:
Corey wanted this to not be just for pumping

Reply:
@coreysprague